src/SprykerSdk/Zed/AppSdk/Business/Configuration/Builder/ConfigurationBuilder.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Zed\AppSdk\Business\Configuration\Builder;
+
+use Generated\Shared\Transfer\AppConfigurationRequestTransfer;
+use Generated\Shared\Transfer\AppConfigurationResponseTransfer;
+
+class ConfigurationBuilder implements ConfigurationBuilderInterface
+{
+    /**
+     * @var array<string>
+     */
+    protected $transferToAsyncApiTypeMap = [
```

Comment:
This is not needed. you can remove. Also the coverage report will show it.

Reply:
I removed it.

Reply:
I can still see it. Either you haven#t committed or it's still there.